spec/factories/users.rb

```diff
@@ -70,6 +70,18 @@
       end
     end
 
+    factory :disabilities_compensation_user, traits: [:loa3] do
+      first_name('Beyonce')
```

Comment:
I feel like having Beyonce test data deserves an acknowledgment of some sort.. so .. 🥇